src/client.rs

```diff
@@ -31,9 +31,11 @@ box_castable! {
     /// A client config being constructed. A builder can be modified by,
     /// e.g. rustls_client_config_builder_load_roots_from_file. Once you're
     /// done configuring settings, call rustls_client_config_builder_build
-    /// to turn it into a *rustls_client_config. This object is not safe
-    /// for concurrent mutation. Under the hood, it corresponds to a
-    /// `Box<ClientConfig>`.
+    /// to turn it into a *rustls_client_config. Alternatively, if an error
```

Comment:
This is following the prevailing style, but I think some paragraphs would be beneficial to split up the text. eg, I would make "Alternatively, if an error..." and "This object is not safe for..." their own paragraphs.

Reply:
Good call. I fixed up the instances being touched in this PR.

I think this repo would benefit from a project-wide refresh of the doc strings but it's a bit too much of a lift for me to start this morning.